lib/config/httpRoutes.js

```diff
@@ -1,125 +1,40 @@
-  {verb: 'get', url: '/users/:_id', controller: 'security', action: 'getUser', infos: {
-    description: 'Retrieve the user definition for a user',
-    parameters: [
-      {
-        description: 'The user ID',
-        in: 'path',
-        name: '_id',
-        required: true,
-        type: 'string'
-      }
-    ],
-    produces: ['application/json'],
-    responses: {
-      '200': {
-        description: 'The user exists and has been retrieved',
-        schema: {
-          type: 'object',
-          properties: {
-            action: {type: 'string', description: 'The action into Kuzzle'},
-            controller: {type: 'string', description: 'The controller into Kuzzle'},
-            error: {type: 'null', description: 'The error object or null'},
-            metadata: {type: 'object', description: 'The Metadatas'},
-            requestId: {type: 'object', description: 'The Request ID'},
-            result: {
-              type: 'object',
-              properties: {
-                _id: {type: 'object', description: 'The user _id'},
-                _source: {
-                  type: 'object',
-                  description: 'The user document source',
-                  properties: {
-                    name: {type: 'string', description: 'The user name'},
-                    profile: {type: 'object', description: 'The profile object'}
-                  }
-                }
-              }
-            },
-            status: {type: 'integer', description: 'The status code, here 200'}
-          }
-        }
-      },
-      '404': {
-        description: 'The user does not exists',
-        schema: {
-          type: 'object',
-          properties: {
-            action: {type: 'string', description: 'The action into Kuzzle'},
-            controller: {type: 'string', description: 'The controller into Kuzzle'},
-            error: {
-              type: 'object',
-              description: 'The error object',
-              properties: {
-                _source: {
-                  type: 'object',
-                  properties: {
-                    _id: {type: 'string', description: 'The given user ID, which does not exist'},
-                    body: {type: 'object', description: 'An emty object, since it is a GET query'}
-                  }
-                },
-                message: {type: 'string', description: 'The error message'}
-              }
-            },
-            metadata: {type: 'object', description: 'The Metadatas'},
-            requestId: {type: 'object', description: 'The Request ID'},
-            result: {type: 'null', description: 'The result, here it is null'},
-            status: {type: 'integer', description: 'The status code, here 404'}
-          }
-        }
-      }
-    }
-  }},
+  {verb: 'get', url: '/roles/:_id', controller: 'security', action: 'getRole'},
+  {verb: 'get', url: '/users/:_id', controller: 'security', action: 'getUser'},
```

Comment:
I don't see the point to keep 2 route with info over 150 without it. But in fact, I don't really matter.